pkg/qemu/qemu.go

```diff
@@ -624,9 +625,44 @@ func Cmdline(ctx context.Context, cfg Config) (exe string, args []string, err er
 				return "", nil, err
 			}
 			logrus.Infof("Using system firmware (%q)", firmware)
+			if *y.Firmware.CompatUEFIViaBIOS && *y.Arch == limayaml.X8664 {
+				firmwareVars, err := getFirmwareVars(exe, *y.Arch)
+				if err != nil {
+					return "", nil, err
+				}
+				logrus.Infof("Using system firmware vars (%q)", firmwareVars)
+				varsFile, err := os.Open(firmwareVars)
+				if err != nil {
+					return "", nil, err
+				}
+				defer varsFile.Close()
+				codeFile, err := os.Open(firmware)
+				if err != nil {
+					return "", nil, err
+				}
+				defer codeFile.Close()
+				downloadedFile, err := os.OpenFile(downloadedFirmware, os.O_CREATE|os.O_WRONLY, 0o644)
```

Comment:
Probably the file name should be changed, as the concatenated file would be incompatible with `QemuEfiCodeFD`

Reply:
Will change this. No objection here. I just re-used current code for quick prototype, but it is obviously not the most straightforward or easy to understand hack.